src/index.js

```diff
@@ -257,7 +278,24 @@ class EventPortal {
       throw new Error(error)
     }
   }
-  
+
+  /**
+  * Return the event version object given the eventVersionId
+  * In this case we don't have the eventID, so we need to get the object directly from the eventVersionID
+  * * Warning * : This is not an offically supported Event Portal API
```

Comment:
Unfortunately, I had to use a "hidden" API to get the Event Version Object. I don't have the parent event id so I had to use the "eventVersions" API to get the event version object directly from the eventVersionId.